src/applications/personalization/profile/components/ProfileSubNavItems.jsx

```diff
@@ -21,31 +33,108 @@ function ProfileSubNavItems({ routes, isLOA3, isInMVI, clickHandler = null }) {
     // mvi check
     return !(route.requiresMVI && !isInMVI);
   });
-  const recordNavUserEvent = () => {
+
+  let notificationSettingsIndex;
+  let paperlessDeliveryIndex;
+  let beforeRoutes;
+  let nestedRoutes;
+  let afterRoutes;
+  if (showPaperlessDelivery) {
```

Comment:
If I understand this correctly, you're separating the menu items into the specific ones that have sub-items and the ones that don't and are either before or after the ones that do. This seems like an approach that won't grow well if we add more nested items. Is it not possible to instead do something like (pseudo-code):
```
filtered.map(route => (
  if route === list_of_routes_with_sub_items {
    <VaSidenavSubmenu>
    }
  else {
    <VaSidenavItem>
  }
))
``
That's very rough, but my point is just that we should be able to iterate through all of them and determine how they should be rendered without creating order-dependent stacks.